packages/direflow-component/src/components/Styled.tsx

```diff
@@ -1,8 +1,10 @@
 import React, { FC, Component, ReactNode, ComponentClass, CSSProperties } from 'react';
 import Style from 'style-it';
 
+type TStyles = string | string[] | CSSProperties | CSSProperties[];
+
 interface IStyled {
-  styles: CSSProperties | CSSProperties[];
+  styles: TStyles;
```

Comment:
since the style-values we get from the imports are strings, we the "Styled" component to support strings as type. otherwise, it won't accept our input anymore.
(still works as expected with strings)